### PR TITLE
f-header@10.2.0 - Add a theme to support header on Search

### PR DESCRIPTION
**Pull request by 1itvinka**
### Added
- Theme to support header on Search

---

**Desktop view:**
Screenshot 2022-07-13 at 10 05 07

**Mobile view:**
Screenshot 2022-07-13 at 09 59 05

## UI Review Checks

- [x] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]
- [x] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [x] Safari
- [x] Firefox

### CODE REVIEW

File: packages/components/organisms/f-header/src/components/Header.vue [hunk 1]
```diff
@@ -173,6 +174,10 @@ export default {
             return this.headerBackgroundTheme === 'transparent' && !this.mobileNavIsOpen;
         },
 
+        showSearchHeader () {
```
Comment (jamieomaguire) [inline packages/components/organisms/f-header/src/components/Header.vue]: I'm not sure if we should add such domain-specific language here. I understand its solving a search problem, but is it possible to re-word it to be a little more generic/reusable?

Comment (1itvinka) [reply packages/components/organisms/f-header/src/components/Header.vue]: happy to rename it, but I'm not sure what could be a good name for this theme. It was confirmed by the design team that this variant will be used only on search.

Comment (ashleynolan) [reply packages/components/organisms/f-header/src/components/Header.vue]: Agree with @jamieomaguire – I'd name it closer to how the theme looks/behaves than the page it's used on. Just as it's only used on Search now, doesn't mean that it'll never be on another page in the future.

Is the main difference in the visual theme that it's orange on narrow screens and white on wide screens?

**Comment by jamieomaguire** [comment]
Do we need to add a visual test to check the new variant?

**Review by jamieomaguire — CHANGES_REQUESTED** [review]
Could we add desktop and mobile visual tests to support the new variant please?

File: packages/components/organisms/f-header/src/components/Header.vue (hunk 1 above)
Comment (1itvinka) [reply packages/components/organisms/f-header/src/components/Header.vue]: It's white on narrow now, and transparent with no gradient or `box-shadow` on wide screens.

Comment (ashleynolan) [reply packages/components/organisms/f-header/src/components/Header.vue]: I'd probably call it something like `disappearingWhite`. As long as you document what it does on narrow and wide in the prop for the theme, then that should be good then.

**Review by jamieomaguire — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**